lib/test_cleanup.rb

```diff
@@ -103,12 +103,13 @@ def cleanup_shared
 
       # Keep all "brew" invocations after HOMEBREW_REPOSITORY operations
       # (which cleans up Homebrew/brew)
-      Tap.names.each do |tap_name|
+      taps_to_remove = Tap.names.map do |tap_name|
```

Comment:
Maybe `.select` would fit here better?

Reply:
It's mapping to the `path` value. It could be a `select` then a `map` but this feels a bit neater.